scripts/linux-x86_64/install.sh

```diff
@@ -59,6 +59,10 @@ then
   exit
 fi
 
+if [ -s /etc/systemd/system/albyhub.service ]; then
+  sudo truncate -s 0 /etc/systemd/system/albyhub.service
+fi
+
 sudo tee -a /etc/systemd/system/albyhub.service > /dev/null << EOF
```

Comment:
I think we only should remove the `-a` here which will overwrite the file and not append

Reply:
Yes, more concise